stories/core/Table.stories.tsx

```diff
@@ -1367,6 +1368,7 @@ export const Full: Story<Partial<TableProps>> = (args) => {
         isSelectable
         isSortable
         isResizable
+        hasColumnManager
```

Comment:
hmm... should we also change width rules here, cause this does not look nice

![image](https://user-images.githubusercontent.com/81580355/160153170-5c3d0b65-222a-425d-8460-cc8819f872e4.png)

Reply:
![margin left screenshot cropped](https://user-images.githubusercontent.com/78564221/160303275-43c82292-3325-4acf-950c-5f79ef8aff19.jpg)

Can we add `margin-left: auto` to the column manager column

Reply:
hmmm... But in thise case `Click` column is left in small width and that empty space is still for nothing, right?
I dont know.. Kinda fixes visually..
@FlyersPh9 opinion?
For context, my screenshot of table and column manager left aligned appears when other columns have certain width rules. Question: should we try fixing column manager placement or just change our examples and set width to auto adjust..?

Reply:
I removed the width rules in the columns for the Column Manager story. If I do that for the Full story, when all of the columns are hidden, I still get the issue.

![image](https://user-images.githubusercontent.com/78564221/160410633-0446c52d-dbe7-4da3-b4ec-6b8dcccaff4c.png)

I think we should try fixing column manager placement

Reply:
I have no idea how we should fix this since columns has static width. I guess in most cases there should be name/id column that can't be removed and it would take remaining space like in `Column Manager` story.

Reply:
> there should be name/id column that can't be removed and it would take remaining space

That makes sense to me.  Named column can be disabled from hiding & take remaining width.  Otherwise with this solution there is a lot of wasted space.
![Screen Shot 2022-03-29 at 3 17 21 PM](https://user-images.githubusercontent.com/849817/160690287-3584a2ac-3c70-4c01-a30e-3fb1420d5a2e.png)

Reply:
So lets add then `Name` column as not configurable in our `Full` story to avoid weird case without any useful column

Reply:
`Full` story changed